typings/axe-core/axe-core.d.ts

```diff
@@ -3,19 +3,27 @@
 // Definitions by: Marcy Sutton <https://github.com/marcysutton>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+export type ImpactValue = "minor" | "moderate" | "serious" | "critical" | "null";
```

Comment:
Gotcha. Maybe when TS 2.0 comes out it would be a good idea to create a 1.8 `.d.ts` file, and make this friendlier to people using 2.0 with `--strictNullChecks`.

For now you can omit the `null` and the only people who might have a problem are those using 2.0 with `--strictNullChecks`.